composer.lock

```diff
@@ -240,20 +255,105 @@
                 }
             ],
             "description": "jQuery UI is a curated set of user interface interactions, effects, widgets, and themes built on top of the jQuery JavaScript Library. Whether you're building highly interactive web applications or you just need to add a date picker to a form control, jQuery UI is the perfect choice.",
+            "support": {
+                "issues": "https://github.com/components/jqueryui/issues",
+                "source": "https://github.com/components/jqueryui/tree/master"
+            },
             "time": "2016-09-16T05:47:55+00:00"
         },
+        {
```

Comment:
Unless all of these other `composer.lock` changes are introduced by adding the `composer/semver` dependency, this is the only appropriate change to this file.

Reply:
If the require is done before even installing it, then it does all the same updates and leaves my composer.lock the same as this version.

Reply:
@asmecher , what is your opinion on PRs and dependency lockfile management?  Obviously, we want to insert a new dependency here, but I don't think we want to generate a new lockfile baseline with a bunch of coincidental updates.